srcpkgs/ctwm/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'ctwm'
+pkgname=ctwm
+version=4.0.2
+revision=1
+build_style=cmake
+configure_args="-DUSE_JPEG=OFF"
+hostmakedepends="flex m4"
+makedepends="libX11-devel libXext-devel libXmu-devel libXt-devel libSM-devel libICE-devel libXpm-devel libXrandr-devel"
+depends="m4"
```

Comment:
is `m4` realy a runtime dependency of `ctwm`

Reply:
Yes, it is used to preprocess config files. I have added m4 and other optional dependencies as build options.

Reply:
Ok. Thank you for clarifying. Looked suspicious.